projects/demo/src/app/app.component.html

```diff
@@ -27,10 +29,10 @@ <h1 *uib-template="'title';let config">{{ config.title }} - DEMO</h1>
   <div class="p-1 filters">
     <h3>Filters</h3>
     <uib-zone id="filters">
-      <div *uib-template="'range';let config" display="Range filter">
+      <ng-template *uib-template="'range';let config" display="Range filter">
```

Comment:
Did you mean `ng-container` rather than `ng-template`?

Reply:
Both works... I change it to `<ng-container>`